nessie/jobs/background_job.py

```diff
@@ -63,10 +55,18 @@ def get_s3_calnet_daily_path(cutoff=datetime.now()):
     return app.config['LOCH_S3_CALNET_DATA_PATH'] + '/daily/' + today_hash + '-' + today
 
 
-def get_s3_asc_daily_path(cutoff=datetime.now()):
+def get_s3_canvas_daily_path():
+    # TODO Temporarily share Data Loch's existing hash algorithm, even if it doesn't match PDG standard practice.
+    # today = localize_datetime(datetime.now()).strftime('%Y-%m-%d')
+    today = localize_datetime(datetime.now()).strftime('%m-%d-%Y')
+    today_hash = hashlib.md5(today.encode('utf-8')).hexdigest()
+    return app.config['LOCH_S3_CANVAS_DATA_PATH_DAILY'] + '/' + today_hash + '-' + today
+
+
+def get_s3_sis_daily_path(cutoff=datetime.now()):
     today = localize_datetime(cutoff).strftime('%Y-%m-%d')
     today_hash = hashlib.md5(today.encode('utf-8')).hexdigest()
-    return app.config['LOCH_S3_ASC_DATA_PATH'] + '/daily/' + today_hash + '-' + today
+    return app.config['LOCH_S3_SIS_DATA_PATH'] + '/daily/' + today_hash + '-' + today
```

Comment:
I reordered methods above – the diff is a bit confusing. Reviewing raw source might be easier.